tests/pip/vagga.yaml

```diff
@@ -85,6 +85,15 @@ containers:
     - !Env { CFLAGS: "-I/usr/include/libxml2 -I/usr/include/libxslt" }
     - !Py3Install [lxml]
 
+  pip-deps:
+    setup:
+    - !Ubuntu focal
+    # Fails with following error if setuptools version is not specified:
+    # error in anyjson setup command: use_2to3 is invalid
+    - !PipConfig
+      pip-deps: [setuptools<58]
+    - !Py3Install [anyjson==0.3.3]
```

Comment:
It's an example of package that can't be installed with `setuptools>=59`. And it isn't trivial to fix the error. Following solution almost works:

```yaml
    - !Sh |
        export PYTHONPATH=/tmp/pip-install
        python3 -m pip uninstall -y setuptools
        python3 -m pip install "setuptools<58.0"
```

but requires to install at least one package before that.

Reply:
But you should check you don't have cached wheel.